templates/zerver/help/include/topic-long-press-menu.md

```diff
@@ -0,0 +1 @@
+1. Press and hold the topic to access the long-press menu.
```

Comment:
Let's change "topic" -> "topic bar"; I think that will be more clear.

Reply:
Yes, I agree. Will do!